.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+name: Tests
+on: [pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install requirements
+        run: pip install flake8 pycodestyle
+      - name: Check syntax
+        run: |
+          flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
```

Comment:
Remove as `flake8 . --count --max-line-length=127 --show-source --statistics` will be enough.

Reply:
done